Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/ModularViewportCameraController.cpp

```diff
@@ -381,7 +380,7 @@ namespace AtomToolsFramework
     {
         if (!m_storedCamera.has_value())
         {
-            m_storedCamera = m_previousCamera;
+            m_storedCamera = m_targetCamera;
         }
 
         const auto angles = AzFramework::EulerAngles(AZ::Matrix3x3::CreateFromQuaternion(worldFromLocal.GetRotation()));
```

Comment:
The AZ Quaternion class has utility functions to retrieve the Euler angles (GetEulerDegrees, GetEulerRadians), no need to convert to a 3x3 matrix first.

Reply:
I am pretty sure I looked into this in in the past and it doesn't give the angles in the format I want (`AzFramework::EulerAngles` is specially written for the camera). This is outside the scope of the review for now so I'd rather leave as-is for now and I can check back on it as part of another PR if necessary.

Reply:
Seems fine to leave it for another PR. It's working, just not the most efficient.

Reply:
For sure, there are just so many versions of combining Euler angles the one used here is very specific (and handles us having Z as up too for example when converting to pitch/yaw/roll).